### ISSUE TITLE
Error building image

### ISSUE THREAD

**Issue by Bram-Zijp** [windmilleng/tilt#1681]
OS: Ubuntu 18.04
Docker: 18.09.6

When building a Dockerfile with the `docker_build('x', '.')` function, I get the following error:
```
╎ Created tarball (size: 3.0 MB)
│ Building image
╎ ERROR IN: [internal] extracting build context
╎   → container_linux.go:345: starting container process caused "process_linux.go:424: container init caused \"rootfs_linux.go:109: jailing process inside rootfs caused \\\"pivot_root invalid argument\\\"\""
ERROR: ImageBuild: failed to resolve dockerfile: executor failed running [copy --unpack /src/context /out/]: exit code: 1
```

Do you guys have any idea what can be the reason for this error?

**Comment by nicks** [comment]
Wow I've never seen this error message before!

Could you try running:

`
DOCKER_BUILDKIT=1 docker build -t x .
`

And see if it reproduces outside tilt? You could also try running `tilt doctor` which will give a bunch of diagnostic info about the environment.

**Comment by eLco** [comment]
I can confirm same issue, running on Mac OS with minikube.
Tilt doctor output:

Tilt: v0.8.7, built 2019-05-23
System: darwin-amd64
---
Docker
- Host: tcp://192.168.64.13:2376
- Version: 1.39
---
Kubernetes
- Env: minikube
- Context: minikube
- Cluster Name: minikube
- Namespace: default
- Container Runtime: docker
- Version: v1.14.2
---
Thanks for seeing the Tilt Doctor!
Please send this info along when filing bug reports. 💗

**Comment by nicks** [comment]
ah, i see, looks like a bug in how minikube interacts with buildkit in the latest version, i think we can fix this pretty easily

**Comment by nicks** [comment]
for reference: https://github.com/kubernetes/minikube/issues/4143

**Comment by Bram-Zijp** [comment]
I also have an MacOS which I used for a longer time. It probably has older versions and it's working fine on this machine. I will post my tilt doctor results of the Ubuntu when I reach home.


Tilt: v0.8.7, built 2019-05-23
System: darwin-amd64
---
Docker
- Host: tcp://192.168.99.100:2376
- Version: 1.38
---
Kubernetes
- Env: minikube
- Context: minikube
- Cluster Name: minikube
- Namespace: default
- Container Runtime: docker
- Version: v1.14.1
---
Thanks for seeing the Tilt Doctor!
Please send this info along when filing bug reports.

**Comment by nicks** [comment]
We'll do a release next week with this fix, thanks for reporting!

**Comment by eLco** [comment]
Thank you, for a quick fix!

**Comment by nicks** [comment]
this should be fixed in v0.8.8, which we just released!
